src/raghvendra/NumberOperationUsingWhile.java

```diff
@@ -0,0 +1,59 @@
+package raghvendra;
+
+public class NumberOperationUsingWhile {
+	
+	void evenNumber(int start,int end) {
+		int index=start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment? same for all other methods